WooCommerce/Classes/ViewRelated/Orders/Payment Methods/PaymentMethodsView.swift

```diff
@@ -130,8 +130,7 @@ struct PaymentMethodsView: View {
             ShareSheet(activityItems: [viewModel.paymentLink].compactMap { $0 } ) { _, completed, _, _ in
                 if completed {
                     dismiss()
-                    viewModel.performLinkSharedTasks()
-                }
+                    viewModel.performLinkSharedTasks()                }
```

Comment:
I think this got modified by mistake!

Reply:
Oops! It must be an oversight when I removed the `dismiss` call I added below previously.